### PR TITLE
zh:announcing-1.0.1

### PR DESCRIPTION
**Pull request by roganw**

### CODE REVIEW

**Comment by rootsongjc** [comment]
@roganw CI failed with the following errors.
```
- ./public/zh/blog/2018/announcing-1.0.1/index.html
  *  100:10: ERROR: htmlParseStartTag: misplaced <body> tag (line 100)
  *  98:12: ERROR: Unexpected end tag : head (line 98)
htmlproofer 3.9.2 | Error:  HTML-Proofer found 2 failures!
```
Please file a commit to fix it.

**Comment by roganw** [comment]
@rootsongjc  Fixed it.

File: content_zh/blog/2018/announcing-1.0.1/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Istio 1.0.1 发布
+description: Istio 1.0.1 补丁版本。
+publishdate: 2018-08-29
+attribution: The Istio Team
+weight: 82
+---
+
+我们高兴地发布了 Istio 1.0.1。请查看 [发行说明](/about/notes/1.0.1/) 来了解和下载更新。
```
Comment (rootsongjc) [inline content_zh/blog/2018/announcing-1.0.1/index.md]: 链接前不需要加空格，请使用使用中文文档的链接。

Comment (roganw) [reply content_zh/blog/2018/announcing-1.0.1/index.md]: updated

**Pull request merged**